index.html

```diff
@@ -15,16 +15,44 @@
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
+
+
 <body>
+
   <header>
     <div class="container">
+      Code Journal
     </div>
   </header>
+
   <main class="container">
-    <div data-view="entry-form"></div>
+    <form data-view="entry-form">
+      <h1>New Entry</h1>
+
+      <div class="row-one">
+        <img src="images/placeholder-image-square.jpg" class="image">
+
+        <div class="row1-col2">
+          <div class="title-label column-half"><label class="title" >Title</label></div>
+          <input type="text" name="title" class="column-half title-label" required="required"/>
+
+          <div class="photo-label column-half"><label for="##" class="">Photo URL</label></div>
```

Comment:
All `label` elements should have a `for` attribute that matches the `id` of an `input`